docs/UserGuide.md

```diff
@@ -1020,18 +1020,18 @@ Some commands require you to include parameters. These parameters are identified
+| [m\| ] | Matriculation ID          | Fixed at 9 | The first letter must be an uppercase 'A', followed by 7 numbers, and end with an uppercase letter.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| [r\| ] | Reflection Group          | 4           | The first letter must be an uppercase 'R', followed by any number.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| [s\| ] | Studio Group              | 4           | The first letter must be an uppercase 'S', followed by any number.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| [s\| ] | Score                     | 7 + 2 decimals | For exam max scores: the input must be a positive integer.<br/> For persons' exam scores: the input must be an integer greater than or equal to zero.                                                                                                                                                                                                                                                                                                                                                                                                                                 |
```

Comment:
Insightful change, we should learn to be more thorough with our casing.

Reply:
👍